.github/scripts/run-codespell.py

```diff
@@ -0,0 +1,40 @@
+"""
+ Copyright (C) 2024 Intel Corporation
+
+ Under the Apache License v2.0 with LLVM Exceptions. See LICENSE.TXT.
+ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+import subprocess  # nosec B404
+import logging
+import sys
+
+logging.basicConfig(
+    level=logging.INFO, format="[%(levelname)s]: [%(asctime)s] %(message)s"
+)
+
+
+def codespell_scan():
+    try:
+        codespell_result = subprocess.run(  # nosec
+            [
+                "codespell",
+                "-H",
+                "--quiet-level=3",
+                "--skip=./.git,./.venv,./.github/workflows/.spellcheck-conf.toml,./.github/scripts/run-codespell.py",
```

Comment:
heh, why skipping itself `./.github/scripts/run-codespell.py"` ?

Reply:
It earlier had some words to ignore and codespell detected them as words to correct even in `./.github/scripts/run-codespell.py`. Since last commit I removed `--ignore-words` flag, I can remove `./.github/scripts/run-codespell.py` from `--skip`. Should I do so?

Reply:
Yes, exactly. you can remove it now.

Reply:
yes, the general rule would be to use ignored-words dictionary, but for `.spellcheck-conf.toml` file we could skip it - it doesn't matter that much

as for `run-codespell.py` - yes, definitely remove from "skipping" - we want to scan as much files, as possible

Reply:
Done

Reply:
OK, the build passed:
https://github.com/oneapi-src/unified-memory-framework/actions/runs/12413379227/job/34655130988?pr=994
![image](https://github.com/user-attachments/assets/176338cb-1971-45e4-9dd6-0a14adbc9f11)